cmd/uid_provider/client/interface.go

```diff
@@ -3,21 +3,21 @@ package client
 import (
 	"context"
 
-	"github.com/inspr/inspr/pkg/api/auth"
+	"github.com/inspr/inspr/pkg/auth/models"
 )
 
 // User defines the information a user contains
 type User struct {
-	UID      string
-	Role     int
-	Scope    []string
-	Password string
+	UID        string
+	Permission []string
```

Comment:
Shouldn't it be named permissions? It just looks a little off to me.

Reply:
Hummmm, I agree